.storybook/stories/playground/index.js

```diff
@@ -25,7 +25,16 @@ export default {
 
 // @todo: Find better way to mock these.
 const config = {
-	allowedVideoMimeTypes: [ 'video/mp4' ],
```

Comment:
Can we just find a better way of mocking these?

Reply:
Suggestions welcome! I haven't spent much time looking into it. Perhaps @wassgha has an idea

Reply:
Resolving as actual mocking should be handled in a separate issue. This PR just aims to fix the broken playground ASAP.